packages/interface/src/features/admin/components/DeployRounds.tsx

```diff
@@ -0,0 +1,135 @@
+import { useRouter } from "next/router";
+import { useState } from "react";
+import { toast } from "sonner";
+import { useAccount } from "wagmi";
+
+import { ImageUpload } from "~/components/ImageUpload";
+import { Steps } from "~/components/Steps";
+import { Form, FormSection, FormControl, Textarea, Select, DateInput } from "~/components/ui/Form";
+import { Heading } from "~/components/ui/Heading";
+import { Input } from "~/components/ui/Input";
+import { RoundSchema, votingStrategyTypes } from "~/features/rounds/types";
+import { useIsCorrectNetwork } from "~/hooks/useIsCorrectNetwork";
+
+import { useDeployRound } from "../hooks/useDeployRound";
+import { EDeployStep } from "../types";
+
+import { DeployRoundsButtons } from "./DeployRoundsButtons";
+import { ReviewDeployRoundDetails } from "./ReviewDeployRoundDetails";
+
+export const DeployRounds = (): JSX.Element => {
+  const router = useRouter();
+
+  const { isCorrectNetwork, correctNetwork } = useIsCorrectNetwork();
+
+  const { address } = useAccount();
+
+  const [step, setStep] = useState<EDeployStep>(EDeployStep.CONFIGURE);
+
+  const create = useDeployRound({
+    onSuccess: () => {
+      router.push(`/`);
+    },
+    onError: (err: Error) =>
+      toast.error("Round deploy error", {
+        description: err.message,
+      }),
+  });
+
+  const { error: createError } = create;
+
+  return (
+    <div>
+      <Heading size="4xl">Deploy Round Contracts</Heading>
+
+      <p className="text-gray-400">These round contracts specify the features for this round.</p>
+
+      <div className="dark:border-lighterBlack rounded-lg border border-gray-200 p-4">
+        <Steps step={step} stepNames={["Configure Contracts", "Review & Deploy"]} />
+
+        <Form
+          defaultValues={{ roundId: "", description: "" }}
+          schema={RoundSchema}
+          onSubmit={(round) => {
+            create.mutate(round);
+          }}
```

Comment:
Don't use arrow functions inside jsx

Reply:
then should we also change the one inside `features/applications/components/ApplicationForm.tsx`?

Reply:
If you see it's used somewhere, it's better to change it because it affects performance.